buildspec.yml

```diff
@@ -37,34 +37,38 @@ phases:
       # tar_name will be something like: sagemaker_mxnet_container-3.0.0.tar.gz
       - python3 setup.py sdist
       - tar_name=$(ls dist/sagemaker_mxnet_container*.tar.gz)
-      - build_dir="docker/$FRAMEWORK_VERSION"
-      - cp $tar_name $build_dir/sagemaker_mxnet_container.tar.gz
-      - cp docker/artifacts/*.py $build_dir
+      - framework_dir="docker/$FRAMEWORK_VERSION"
+      - cpu_dir=$framework_dir/py$CPU_PY_VERSION
+      - gpu_dir=$framework_dir/py$GPU_PY_VERSION
+      - cp $tar_name $cpu_dir/sagemaker_mxnet_container.tar.gz
+      - cp $tar_name $gpu_dir/sagemaker_mxnet_container.tar.gz
+      - cp docker/artifacts/*.py $cpu_dir
+      - cp docker/artifacts/*.py $gpu_dir
 
       - build_id="$(echo $CODEBUILD_BUILD_ID | sed -e 's/:/-/g')"
       - prev_dir=$(pwd)
-      - cd $build_dir
 
       # build cpu image
+      - cd $cpu_dir
       - cpu_dockerfile="Dockerfile.cpu"
       - CPU_TAG="$FRAMEWORK_VERSION-cpu-py$CPU_PY_VERSION-$build_id"
 
-      - docker build -f py$CPU_PY_VERSION/$cpu_dockerfile --build-arg framework_support_installable=$tar_name -t $PREPROD_IMAGE:$CPU_TAG .
+      - docker build -f $cpu_dockerfile --build-arg framework_support_installable=$tar_name -t $PREPROD_IMAGE:$CPU_TAG .
```

Comment:
This docker build has framework_support_installable, but the $gpu_dockerfile build doesn't.

Afaik, this isn't necessary in either of them, is it?